Holiday Provider for Bosnia

TheAdnan opened the pull request.

stelgenhof commented:
Thanks for the PR! According to my research, these are not all the official Bosnian holidays, are they? For example, I saw there is a second New Years Day and a second Labour Day. Do you maybe have a good resource that explains the official/national holidays?

Cheers! Sacha

TheAdnan commented:
Hey Sacha, thanks for the info, I thought it was by default that the second New Years Day and Labour day were holidays, my bad.
I've fixed it, and I've also added the Orthodox Christmas Day.
Let me know if there's any other issue.

Cheers, Adnan  :tulip:

stelgenhof commented on src/Yasumi/Provider/Bosnia.php:
```diff
@@ -0,0 +1,98 @@
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in Croatia.
```
You might want to correct this :) Please also check everywhere else in all the files if there is anything that needs to be changed.

stelgenhof commented on src/Yasumi/Provider/Bosnia.php:
```diff
@@ -0,0 +1,98 @@
+/**
+ * Provider for all holidays in Croatia.
+ *
+ * @author Karlo Mikus <contact@karlomikus.com>
```
You can change it to your name :)

stelgenhof commented on src/Yasumi/Provider/Bosnia.php:
```diff
@@ -0,0 +1,98 @@
+     * Code to identify this Holiday Provider. Typically this is the ISO3166 code corresponding to the respective
+     * country or sub-region.
+     */
+    const ID = 'BS';
```
According to https://en.wikipedia.org/wiki/ISO_3166-2:BA, the ISO3166-2 code for Bosnia and Herzegovina is 'BA'.

stelgenhof commented:
I think it looks good now. However, due to the various religions and various jurisdictions in Bosnia, maybe other holidays need to be added. 

Source: https://en.wikipedia.org/wiki/Public_holidays_in_Bosnia_and_Herzegovina

TheAdnan commented:
Oh sorry, I'll keep that in mind next time :smiley:
